src/main/java/org/victorrobotics/frc/dtlib/actuator/motor/ctre/DTTalonFX.java

```diff
@@ -101,8 +105,13 @@ public void configPIDintegralZone(double iZone) {
     }
 
     @Override
-    public void configCurrentLimit(SupplyCurrentLimitConfiguration maxCurrent) {
-        internal.configSupplyCurrentLimit(maxCurrent);
+    public void configCurrentLimit(int maxSupplyCurrent) {
+        internal.configSupplyCurrentLimit(
+                new SupplyCurrentLimitConfiguration(true, maxSupplyCurrent, maxSupplyCurrent, 0));
+    }
+
+    public void configCurrentLimit(SupplyCurrentLimitConfiguration currentConfig) {
```

Comment:
This method could take the limits directly and construct the `SupplyCurrentLimitConfiguration` object.

Reply:
True, good idea